project/ScriptCommands.scala

```diff
@@ -81,23 +82,28 @@ object ScriptCommands {
       publishTo in Global := Some("sonatype-releases" at "https://oss.sonatype.org/service/local/staging/deploy/maven2"),
       credentials in Global += Credentials(Path.userHome / ".credentials-sonatype"),
       pgpPassphrase in Global := Some(Array.empty)
-    ) ++ enableOptimizerNewFlags
+    ) ++ enableOptimizer
   }
 
-  private[this] def setup(name: String)(f: Seq[String] => Seq[Setting[_]]) =
-    Command.args(name, name) { case (state, seq) => Project.extract(state).append(f(seq) ++ resetLogLevels, state) }
+  def enableOptimizerCommand = setup("enableOptimizer")(_ => enableOptimizer)
+
+  private[this] def setup(name: String)(f: Seq[String] => Seq[Setting[_]]) = Command.args(name, name) { case (state, seq) =>
```

Comment:
I wrote the exact same code in my personal project the other day. I wish it was provided by sbt by default!